torch/csrc/jit/passes/onnx.cpp

```diff
@@ -117,7 +120,29 @@ void ToONNX(std::shared_ptr<tracer::TracingState>& state) {
     }
   };
 
-  auto callPySymbolic = [&](PythonOp* op) {
+  auto callPySymbolicFunction = [&](Node* n) {
+    // The idea is delegate as much of the actual argument massaging to
+    // Python as possible
+
+    // TODO: Arguably this should use the converters, but how do you apply
```

Comment:
I am very fine not using converters unless they are something we use in tons of places. It just makes things harder to follow.